source/documentation/modules/color.liquid

```diff
@@ -0,0 +1,834 @@
+---
+title: sass:color
+---
+
+{% render 'documentation/snippets/built-in-module-status' %}
+
+{% function 'color.adjust($color,!!!  $red: null, $green: null, $blue: null,!!!  $hue: null, $saturation: null, $lightness: null,!!!  $whiteness: null, $blackness: null,!!!  $alpha: null)', 'adjust-color(...)', 'returns:color' %}
+{% compatibility '1.28.0' false, null, false, '$whiteness and $blackness' %}{% endcompatibility %}
+
+Increases or decreases one or more properties of `$color` by fixed amounts.
+
+Adds the value passed for each keyword argument to the corresponding property
+of the color, and returns the adjusted color. It's an error to specify an RGB
+property (`$red`, `$green`, and/or `$blue`) at the same time as an HSL
+property (`$hue`, `$saturation`, and/or `$lightness`), or either of those at
+the same time as an [HWB][] property (`$hue`, `$whiteness`, and/or
+`$blackness`).
+
+[HWB]: https://en.wikipedia.org/wiki/HWB_color_model
+
+All optional arguments must be numbers. The `$red`, `$green`, and `$blue`
+arguments must be [unitless][] and between -255 and 255 (inclusive). The
+`$hue` argument must have either the unit `deg` or no unit. The `$saturation`,
+`$lightness`, `$whiteness`, and `$blackness` arguments must be between `-100%`
+and `100%` (inclusive), and may not be unitless. The `$alpha` argument must be
+unitless and between -1 and 1 (inclusive).
+
+[unitless]: /documentation/values/numbers#units
+
+See also:
+
+* [`color.scale()`](#scale) for fluidly scaling a color's properties.
+* [`color.change()`](#change) for setting a color's properties.
+
+{% codeExample 'adjust-color', false %}
```

Comment:
After the merge with `main`, most examples are broken in the preview. Because the contents of `{% function %}` are parsed as markdown, there's no simple way to parse `{% codeExample %}` as plain liquid when it's inside a `{% function %}`. IMO we should adjust `codeExample` to work with md as much possible (like we did [here](https://github.com/oddbird/sass-site/pull/51/commits/b7154ddcd0ce86aab7aec2ea239ee3a75cc975d2)), but we could also make markdown parsing optional in `{% function %}` and use explicit `{% markdown %}` blocks. Thoughts?

Reply:
Done in https://github.com/oddbird/sass-site/pull/54/commits/549f3d04506cf9461d1552fe86ecb1ff0c1d52fb